src/pages/Login/Login.tsx

```diff
@@ -44,6 +44,7 @@ const Login = () => {
         //clear the input after successful submit
         emailRef.current.value = '';
         passwordRef.current.value = '';
+        navigate('/dashboard', { replace: true });
```

Comment:
![image](https://github.com/GenericSoft/GDR-Generic-Desk-Reservation/assets/31138395/7de77ab0-13da-4da2-936d-5a0b9433e71b)

Check if the user is logged in first, because currently you can open dashboard page without login.